api/src/infrastructure/kubernetes/payloads.rs

```diff
@@ -63,15 +63,14 @@ macro_rules! secret_name_from_name {
 }
 
 /// Creates a JSON payload suitable for [Kubernetes' Namespaces](https://kubernetes.io/docs/tasks/administer-cluster/namespaces/)
-pub fn namespace_payload(app_name: &String) -> String {
+pub fn namespace_payload(app_name: &String) -> Value {
```

Comment:
See comment above: I would prefer to return [`Namespace`](https://docs.rs/k8s-openapi/0.11.0/k8s_openapi/api/core/v1/struct.Namespace.html) instead of `Value`.